client/src/pages/component/ContactUs/ContactUs.js

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import "./ContactUs.css";
+import { FaGithub, FaLinkedin } from "react-icons/fa";
+function ContactUs() {
+	return (
+		<>
+			<div className="contactUs-container">
+				<div className="contactUs-main-card">
+					<p>Contact Us</p>
```

Comment:
It's much better to use the heading elements for headings - it's more semantic and it helps accessibility (screen readers etc) and e.g. how Google would index the page. Given that this is the main heading for the page, consider using `<h1>` here.